src/App.js

```diff
@@ -1,12 +1,12 @@
+import Data from "./Data/cardData.json"
```

Comment:
could you start with a lower letter for the standard variable and also remember to start your directories name with lowercase
```suggestion
import data from "./Data/cardData.json"
```